core-server/src/main/java/org/glassfish/jersey/server/ServerRuntime.java

```diff
@@ -669,11 +669,18 @@ public OutputStream getOutputStream(final int contentLength) throws IOException
 
             } catch (final Throwable ex) {
                 if (response.isCommitted()) {
-                    /**
-                     * We're done with processing here. There's nothing we can do about the exception so
-                     * let's just log it.
-                     */
-                    LOGGER.log(Level.SEVERE, LocalizationMessages.ERROR_WRITING_RESPONSE_ENTITY(), ex);
+                    if (ex.getCause() instanceof IOException) {
```

Comment:
The `ex instanceof IOException` is relevant here too.

Reply:
Not sure if you want to check all nested causes and suppressed throwables for IOException too?